source/help/getting-started/organizing.rst

```diff
@@ -42,151 +42,150 @@ As you grow further, you can combine the hierarchies, with formats like ``[SUB-T
 
-When it's time to meet, your team can click the conference link to connect. 
+When it's time to meet, your team can click the conference link to connect.
 
-Sub-Teams 
+Sub-Teams
 ~~~~~~~~~~~~~~~~~~
 
 Sub-teams can include people from the same discipline, project teams, people with the same manager or other groups brought together for a shared purpose::
 
-	Examples: Developers, Marketers, Offsite Organizing Committee, SusanK's Directs 
+	Examples: Developers, Marketers, Offsite Organizing Committee, SusanK's Directs
 
 As sub-teams grow beyond a manageable size for one channel, they can sub-divide::
 
 	Examples: US: Developers, UK: Developers, SusanK's Directs, SusanK's Extended Directs
 
-Projects 
+Projects
 ~~~~~~~~~~~~~~~~~~
 
-Project channels discuss how groups of people can come together to achieve specific outcomes:: 
+Project channels discuss how groups of people can come together to achieve specific outcomes::
 
-	Examples: Logo Design, Localization, Product Launch 
+	Examples: Logo Design, Localization, Product Launch
 
-Projects are often private groups rather than public channels and are often used to organize a small team around a project brought up in a larger channel. The Project Channel is used to do detailed work, and updates are communicated back to larger channels in many instances. 
+Projects are often private channels rather than public channels and are often used to organize a small team around a project brought up in a larger channel. The Project Channel is used to do detailed work, and updates are communicated back to larger channels in many instances.
 
-Location Channels 
+Location Channels
 ~~~~~~~~~~~~~~~~~~
 
 If your teams are in different buildings, cities or regions, you can create channels to help people coordinate meetings and get-togethers::
 
 	Examples: Building 10, Palo Alto, Toronto, Delaware
 
-This helps share announcements and discussions relevant to only those locations. 
+This helps share announcements and discussions relevant to only those locations.
 
-Data Channels 
+Data Channels
 ~~~~~~~~~~~~~~~~~~
 
-Sometimes you want to setup integrations to automatically bring data into certain channels. Information like Twitter updates, new or updated support tickets or bug reports, or mentions of your company name in the news can all be made available in channels your team choose to monitor. Some people might use these channels like a daily newspaper, reading about everything that's happened since the last day. 
+Sometimes you want to setup integrations to automatically bring data into certain channels. Information like Twitter updates, new or updated support tickets or bug reports, or mentions of your company name in the news can all be made available in channels your team choose to monitor. Some people might use these channels like a daily newspaper, reading about everything that's happened since the last day.
```

Comment:
"setup" > "set up" ?

"...your team choose to monitor." > "...your team chooses to monitor." ?

"...since the last day." > "...since the previous day." ?